changelogs/unreleased/gh-1189-index_opts.covers-and-format-layout-attribute.md

```diff
@@ -0,0 +1,4 @@
+## feature/box
+
+* `index_opts.covers` option and space format now allow specifying per-column
+  'layout' attribute.
```

Comment:
Let's split into adding to options to index covers and to format.

I guess it is better to add changelog in appropriate commit in EE as CE does not support either.